docs/user/gateways.md

```diff
@@ -7,6 +7,15 @@ Both the traces and the metrics feature are based on a gateway, which gets provi
 
```

Comment:
but which one without breaking existing URLs? It can be 06, what is the value?I think we should get rid of the numbers as they are in the URL which was never the idea behind..

Reply:
Yep they no longer add much value after we got `_sidebar.md` files.
If we ever have very many documents in a file, numbering might help keep the overview by forcing an order, but I'm not sure that will ever be necessary.

Reply:
I understand, but they the numbers are rendered in the URL... so I prefer to get rid of them in future